wallet/invoices.c

```diff
@@ -264,6 +263,7 @@ const struct invoice *invoices_create(struct invoices *invoices,
 		sqlite3_bind_null(stmt, 4);
 	sqlite3_bind_text(stmt, 5, label, strlen(label), SQLITE_TRANSIENT);
 	sqlite3_bind_int64(stmt, 6, expiry_time);
+	sqlite3_bind_text(stmt, 11, b11enc, strlen(b11enc), SQLITE_TRANSIENT);
```

Comment:
The index should be 7.  The 7th `?` in the string is the one that gets replaced here.

Reply:
Sorry, pushed this by accident.